doc/user/physics_coupling.rst

```diff
@@ -0,0 +1,82 @@
+**********************
+Physics Coupling
+**********************
+
+Loose Coupling
+----------------
+ARMI supports loose and tight coupling. Loose coupling is interpreted as one-way coupling between physics for a single time node. For example, a power distribution in cycle 0 node 0 is used to calculate a temperature distribution in cycle 0 node 0. This temperature is then used in cycle 0 node 1 to compute new cross sections and a new power distribution. This process repeats itself for the lifetime of the simulation. 
+
+.. image:: images/looseCouplingIllustration.png
+    :width: 800
+    :alt: Loose coupling illustration.
+
+Loose coupling is enabled by default in ARMI simulations.
+
+Tight Coupling
+-----------------
+Tight coupling is interpreted as two-way communication between physics within a given time node. Revisiting our previous example, enabling tight coupling results in the temperature distribution being used to generate updated cross sections (new temperatures induce changes such as Doppler broadening feedback) and ultimately an updated power distribution. This process is repeated iteratively until a numerical convergence criteria is met.
```

Comment:
super minor point but this would be better if it were line wrapped rather than just one huge line.